common/lib/xmodule/xmodule/modulestore/xml_importer.py

```diff
@@ -284,14 +286,16 @@ def import_static(self, data_path, dest_id):
         )
         if self.do_import_static:
             if self.verbose:
-                log.debug("Importing static content and python library")
+                log.debug("Course import %s: Importing static content and python library", self.target_id)
```

Comment:
why is this a debug log ?

Reply:
use `strings and use log.info that the course import is starting to import static content.